### PR TITLE
Add wadl.top

### PR DESCRIPTION
**Pull request by FPVogel**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

 We are WebWaddle Ltd. And were recently formed by an entertainment technology company (https://glander-vt.de/), whose customers are currently being migrated as an existing customer base. Formerly we mainly provided ticketing solutions web-wise, but now are also building up a general hosting infrastructure. I am the appointed Director of WebWaddle Ltd.

Organization Website: 
 https://webwaddle.com/

Reason for PSL Inclusion
====

  We are currently working on creating webhosting packages that don't require an own ICANN-registered domain name. As such we have registered the wadl.top domain name (shortened from the Waddle-part of the Company name) 

We provide each customer with a unique subdomain of `wadl.top`

We'd like to have accurate handling by other software, hence this pull request. We would namely like to achieve:

- Appropriate cookie handling
- Appropriate highlighting in the browser search bar
- isolation between our customer's website

Number of users this request is being made to serve: around 1k initially by migrating customers, are expecting about 10k within the next two years.

DNS Verification via dig
=======

```
λ ~/ dig @1.1.1.1 +short TXT _psl.wadl.top
"https://github.com/publicsuffix/list/pull/1924"
```

Results of Syntax Checker (`make test`)
=========

```
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

### CODE REVIEW

**Comment by FPVogel** [comment]
Thanks @simon-friedberger for adding the tag and bringing this to our attention,
the domain has now been renewed for an additional 2 years (meaning it will now be valid for close to 3 years) and will then auto-renew every two years.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15290,6 +15290,10 @@ reserve-online.com
 bookonline.app
 hotelwithflight.com
 
+// WebWaddle Ltd: https://webwaddle.com/
+// Submitted by Merlin Glander <merlin@webwaddle.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:15294]: If possible, please use a non-personal e-mail so it stays reachable.

**Comment by simon-friedberger** [comment]
* [x] wadl.top expires 2027-01-24
* [x] DNS _psl entries
* [x] Tests pass
* [x] Sorting according to guidelines
* [x] Reasoning/Organization description

File: public_suffix_list.dat (hunk 1 above)
Comment (FPVogel) [reply public_suffix_list.dat]: changed now to hostmaster address

**Review by simon-friedberger — APPROVED** [review]

**Comment by FPVogel** [comment]
Hello everybody,
I understand this is a volunteer-driven project, so I don't want to cause any hassle or anything, but is there anything we can do from our end to move this along?

It's now been two months since the PR was reviewed, so it'd be great to know when we can expect it being merged.

All the best from Germany,
Merlin

**Pull request merged**